schemas/recipes_schema.json

```diff
@@ -31,14 +30,26 @@
       },
       "required": ["id"],
       "additionalProperties":false
-    },
+	},
+	"textId_count":{
```

Comment:
why is that needed ?

Reply:
Metadata is no longer required, so a recipe output that just used a textid and count as params was needed

Reply:
Ok that will solve itself once that is switched back to integer ids.